docker-compose-coverage.yaml

```diff
@@ -0,0 +1,63 @@
+version: '3.3'
+
+services:
+  graphdb:
+    image: docker-registry.ontotext.com/graphdb:${GDB_VERSION}
+    environment:
+      GDB_JAVA_OPTS: >-
+        -Dgraphdb.workbench.importDirectory=/opt/home/import-data/
+        -Dgraphdb.jsonld.whitelist=https://w3c.github.io/json-ld-api/tests/*
+        -Dgraphdb.stats.default=disabled
+        -Dgraphdb.foreground=
+        -Dgraphdb.logger.root.level=ERROR
+    ports:
+      - '7200:7200'
+    volumes:
+      - ./e2e-tests/fixtures/graphdb-import:/opt/home/import-data/
+      - ./e2e-tests/fixtures/graphdb.license:/opt/graphdb/home/conf/graphdb.license
+    networks:
+      - single-spa-network
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f -H 'Accept: text/html' ${GRAPHDB_URL}/index.html"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  workbench:
+    build:
+      context: .
+      dockerfile: root-config.Dockerfile
```

Comment:
I now see that this Dockerfile is not renamed, but I remember a discussion that we must rename it to workbench.Dockerfie or application.Dockerfile or similar, because `root-config` is not appropriate and doesn't make sence in this context

Reply:
renamed